python/cuml/preprocessing/model_selection.py

```diff
@@ -24,6 +24,109 @@
 from typing import Union
 
 
+def stratify_split(X, y, n_train, n_test, x_numba, y_numba, random_state):
```

Comment:
```suggestion
def _stratify_split(X, y, n_train, n_test, x_numba, y_numba, random_state):
```
This function seems for internal purposes, so using a leading underscore would be a good idea